example-get-started/code/README.md

```diff
@@ -33,10 +33,17 @@ $ source .env/bin/activate
 $ pip install -r src/requirements.txt
 ```
 
+Note that dvc is not installed by the requirements file, 
+as it is supposed to be used as a global tool like git. If dvc is not installed,
+use the following command to install it:
+```console
```

Comment:
it's better to point to the install page, let users decide how they want DVC to be installed